compiler/extccomp.nim

```diff
@@ -82,7 +82,7 @@ compiler gcc:
     includeCmd: " -I",
     linkDirCmd: " -L",
     linkLibCmd: " -l$1",
-    debug: "",
+    debug: " -g ",
```

Comment:
-g is a compiler flag for gcc to add debug information to the build. Without it, you can't use breakpoints in the sourcecode in debuggers like gdb. You would have to debug on the binary.